doc/getting-started/sample-app.rst

```diff
@@ -500,30 +463,22 @@ Install and Run ACRN on the Target System
    kernel Debian packages using these commands::
 
      cd ~/acrn-work
+     cp ./acrn-hypervisor*.deb ./*acrn-service-vm*.deb /tmp
      sudo apt purge acrn-hypervisor
-     sudo apt install ./acrn-my_board-MyConfiguration*.deb
-     sudo apt install ./*acrn-service-vm*.deb
+     sudo apt install /tmp/acrn-hypervisor*.deb  /tmp/*acrn-service-vm*.deb
 
-#. Enable networking services for sharing with the HMI User VM::
+#. Enable networking services for sharing with the HMI User VM:
```

Comment:
You need the double colons to format the next block as a code block
```suggestion
#. Enable networking services for sharing with the HMI User VM::
```

Reply:
Thanks! I'll return the line to its original form.